plugins/woocommerce-beta-tester/userscripts/wc-live-branches.user.js

```diff
@@ -106,10 +106,11 @@
 		} else {
 			// TODO: Fetch the list of feature flags dynamically from the API or something.
 			const featureFlags = [
-				'async-product-editor-category-field',
-				'launch-your-store',
-				'minified-js',
-				'product-custom-fields',
+				'pattern-toolkit-full-composability',
+				'printful',
+				'product-editor-template-system',
+				'product-pre-publish-modal',
+				'remote-error-logging',
 				'settings',
```

Comment:
Updated the feature flags to match the ones that are currently disabled in core.json (production environment). Ideally, we should have a way to sync these two files automatically.

Reply:
Yeah, that could be a good way to keep the two files in sync.